src/tests/utilities/test-utils.ts

```diff
@@ -1,144 +1,171 @@
+  export const mockTimebankPersons = (statusCode: number, body: any) => {
+    jest.spyOn(personsClient, "listPersons")
+      .mockReturnValueOnce(createIncomingMessage(statusCode, body));
   };
 
   /**
-   * Timebank total time entries mock
+   * Configures mock listPersonTotalTime response
+   * 
+   * @param statusCode API response statusCode
+   * @param body API response body
    */
-  export const mockTotalTimeEntries = () => {
-    jest.spyOn(timebankClient, "timebankControllerGetTotal")
-      .mockReturnValueOnce(Promise.resolve({ response: message, body: timeTotalsMock1 }))
-      .mockReturnValueOnce(Promise.resolve({ response: message, body: timeTotalsMock2 }))
-      .mockReturnValueOnce(Promise.resolve({ response: message, body: timeTotalsMock3 }));
+  export const mockTimebankPersonTotalTimes = (statusCode: number, body: any) => {
+    const personTotalTimesSpy = jest.spyOn(personsClient, "listPersonTotalTime");
+    for (let i = 0; i < body.length; i++) {
+      personTotalTimesSpy.mockReturnValueOnce(createIncomingMessage(statusCode, body[i]));
+    }
   };
 
   /**
-  * Slack post message mock data
-  */
-  export const mockSlackPostMessage = () => {
-    jest.spyOn(slackUsersClient.chat, "postMessage").mockImplementation(() => Promise.resolve(slackPostMessageMock));
+   * Configures mock fetch to Forecast API to return given data.
+   * Has to always return KeycloakMock access token first during handler tests.
+   * 
+   * @param statusCode API response statusCode
+   * @param body API response body
+   */
+  export const mockForecastResponse = (statusCode: number, body: any, keycloakMock: boolean) => {
+    const fetchSpy = jest.spyOn(mockedFetch, "fetch");
+    if (keycloakMock) {
+      fetchSpy.mockReturnValueOnce(mockAccessToken());
+    }
+    if (statusCode !== 200) {
+      fetchSpy.mockReturnValueOnce(createResponse(statusCode, body));
+    }
+    for (let i = 0; i < body.length; i++) {
+      fetchSpy.mockReturnValueOnce(createResponse(statusCode, body[i]));
+    }
   };
 
   /**
-  * Slack post message mock error
-  */
-  export const mockSlackPostMessageError = () => {
-    jest.spyOn(slackUsersClient.chat, "postMessage").mockImplementation(() => Promise.resolve(slackPostMessageError));
+   * Configures mock list response 
+   * 
+   * @param mockData response mockData
+   */
+  export const mockSlackUsers = (mockData: any) => {
+    jest.spyOn(slackUsersClient.users, "list").mockReturnValueOnce(Promise.resolve(mockData));
   };
 
   /**
-   * Timebank total custom time entries mock
+   * Configures mock postMessage response
    * 
-   * @param mockData custom total time entries data
+   * @param mockData response mockData 
    */
-  export const mockTotalTimeEntriesCustom = (mockData: any[]) => {
-    jest.spyOn(timebankClient, "timebankControllerGetTotal")
-      .mockReturnValueOnce(Promise.resolve({ response: message, body: mockData }));
+  export const mockSlackPostMessage = (mockData: any) => {
```

Comment:
You could look into how this function is used to see if the mockData could be typed